engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -5,14 +5,15 @@ import java.util.concurrent.{ExecutorService, Executors}
 import java.util.{Date, UUID}
 import javax.sql.rowset.serial.SerialClob
 
+import com.mysql.jdbc.exceptions.MySQLTransactionRollbackException
```

Comment:
I thought this became a more generic exception type in one of Khalid's PRs that was merged?